scripts/gulpfiles/build_tasks.js

```diff
@@ -184,6 +183,8 @@ return ${namespace};
  *     blockly_compressed.js
  */
 function buildCompressed() {
+  delete require.cache[require.resolve('../../package.json')]
+  var packageJson = require('../../package.json');
```

Comment:
Is this necessary only if requiring within a function? I ask because I recently added a script for deploy demos that also uses require package.json to get version. 

Also, although this is only 2 lines, I'm thinking we might want to consider also making a helper function for gulp scripts to get the package version.

Reply:
If you are not changing the version number in your script I *think* you should be fine. The problem I was running into was when I updated the version number in my script and then tried to build it would use the old cached version number in `blockly_compressed.js`.

Reply:
Moved this into its own function. It is used across multiple files so added a helper function file.